yt_dlp/extractor/iprima.py

```diff
@@ -176,7 +197,7 @@ def _real_extract(self, url):
         final_result = self._search_json_ld(webpage, video_id, default={})
         final_result.update({
             'id': video_id,
-            'title': title,
+            'title': final_result.get('title', title),
```

Comment:
```suggestion
            'title': final_result.get('title') or title,
```

Reply:
[ie/IPrima] Using "or" instead of get() default.